enhancements/dns/forward-dns-over-tls.md

```diff
@@ -243,7 +243,13 @@ for TLS configuration.
 Implementing this enhancement requires changes in the following repositories:
 
 openshift/api
+* [Addition of DNS-over-TLS Types](https://github.com/openshift/api/pull/1110)
+
 openshift/cluster-dns-operator
+* [Primary Implementation](https://github.com/openshift/cluster-dns-operator/pull/314)
+
+openshift/kubernetes
+* [Admission Check](https://github.com/openshift/kubernetes/pull/1247)
```

Comment:
<https://github.com/openshift/kubernetes/pull/1247> merged after the release-4.11 branch was cut, so the change is in <https://github.com/openshift/kubernetes/blob/release-4.12/openshift-kube-apiserver/admission/customresourcevalidation/dns/validate_dns.go> but not in <https://github.com/openshift/kubernetes/blob/release-4.11/openshift-kube-apiserver/admission/customresourcevalidation/dns/validate_dns.go>.  Please make sure this is clear in the enhancement document.

Reply:
Fixed!